imap/jmap_calendar.c

```diff
@@ -5522,19 +5538,24 @@ static void setcalendarevents_update(jmap_req_t *req,
     }
 
     if (jmap_is_using(req, JMAP_CALENDARS_EXTENSION)) {
-        struct buf blobid = BUF_INITIALIZER;
-        if (jmap_encode_rawdata_blobid('I', mailbox_uniqueid(mbox),
-                    mbox->i.last_uid, NULL, req->userid, NULL, NULL, &blobid)) {
-            json_object_set_new(update, "blobId",
-                                json_string(buf_cstring(&blobid)));
-        }
-        buf_reset(&blobid);
-        if (jmap_encode_rawdata_blobid('I', mailbox_uniqueid(mbox),
-                    mbox->i.last_uid, NULL, NULL, NULL, NULL, &blobid)) {
-            json_object_set_new(update, "debugBlobId",
-                                json_string(buf_cstring(&blobid)));
+        struct index_record record;
```

Comment:
Should we create a function to calculate the 2 blobIds from a mailbox/UID tuple and add them to a JSON object?  The code for create and update is essentially duplicated.

Reply:
yes, good idea. done.